### PR TITLE
Implemented improvements. Made breaking changes to format and routes.

### PR DESCRIPTION
**Pull request by tmathews**
@lostlevels

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,91 +6,81 @@ var debug      = require('debug')('zipper')
...
   }
 
   var id = uuid()
-    , url = (req.secure ? 'https://' : 'http://') + path.join(req.headers.host, 'download', id)
   map[id] = obj
-
   res.status(200).json({
-    url: url
+    url: req.protocol + '://' + [req.headers.host, 'download', id].join('/')
   })
```
Comment (lostlevels) [inline index.js]: No more `/download` route right?

**Comment by lostlevels** [comment]
Other than that, looks good.

File: index.js (hunk 1 above)
Comment (tmathews) [inline index.js]: Yea, forgot to remove this.

**Comment by tmathews** [comment]
Should be G2G.

**Pull request merged**